docs/use-cases-periodic.md

```diff
@@ -1,18 +1,47 @@
 ---
 id: use-cases-periodic
-title: Periodic Execution (aka Distributed Cron)
+title: Periodic Execution
 sidebar_label: Periodic Execution
 ---
 
-Periodic execution, frequently referred to as distributed cron, is when you execute business logic periodically. The advantage of Temporal for these scenarios is that it guarantees execution, sophisticated error handling, retry policies, and visibility into execution history.
+## Motivation
 
-Another important dimension is scale. Some use cases require periodic execution for a large number of entities.
-At Uber, there are applications that create periodic workflows per customer.
-Imagine 100+ million parallel cron jobs that don't require a separate batch processing framework.
+Periodic execution of business logic, also known as a **distributed cron** engine.
 
-Periodic execution is often part of other use cases. For example, once a month report generation is a periodic service orchestration. Or an event-driven workflow that accumulates loyalty points for a customer and applies those points once a month.
+Typical examples of periodic service orchestration include:
```

Comment:
I think this change is actually not accurate as some of the listed cases are periodic execution but not all of them are periodic orchestration.